docs/testing.md

```diff
@@ -47,3 +47,26 @@ In order to run these tests, the following steps need to be made:
 
 List of all test scenarios prepared for this service is available at
 <https://github.com/RedHatInsights/insights-behavioral-spec#ccx-notification-service>
+
+## End-to-end test
+
+If you want to test from an e2e perspective:
+
+1. Use `molodec` to generate and upload an archive for a given cluster.
+1. Use `molodec` again, but define a different set of rules to hit with the
+    generated archive, using the `-r` flag (please, check the `molodec` help
+    output for further info).
+1. Check if the notification is properly received
+
+### ServiceLog case
+
+If you are trying to test the e2e notification in Service Log,
+you should pay attention in the second step described above. The new rules
+to be hit by the archive should contain an error key that is tagged with one
+of the configured tags for the notification.
+
+Right now, it only notified about managed (`osd_customer`) tagged rules,
```

Comment:
I'd add a link to https://github.com/RedHatInsights/ccx-notification-service/blob/master/config.toml#L37 so that the reader doesn't get lost thinking "where are those tags configured?"

Reply:
Good point